scanner/src/main/kotlin/storages/ClearlyDefinedStorage.kt

```diff
@@ -92,10 +91,9 @@ private fun findScanCodeVersion(tools: List<String>, coordinates: ClearlyDefined
 }
 
 /**
- * Return an empty success result for the given [id].
+ * Return an empty success result.
  */
-private fun emptyResult(id: Identifier): Result<ScanResultContainer> =
-    Success(ScanResultContainer(id, listOf()))
+private fun emptyResult(): Result<List<ScanResult>> = Success(emptyList())
```

Comment:
The empty success result can now be a constant. As it is used at multiple places, it could even be moved to the companion object of `ScanResultStorage` and referenced everywhere.

Reply:
I have made them constants for now. I like the idea of having constants in `ScanResultsStorage`, but then it should also be done for `Success(Unit)` and `Success(emptyMap()`, I quickly tried it, but wasn't too happy with the result, so I'd prefer if this was addressed separately. Would you like to do this?

Reply:
Of course, this can be addressed separately.